web/src/pages/insights/SummaryTable.tsx

```diff
@@ -0,0 +1,294 @@
+            {summary.latest_snv_es_index?.timestamp}
+            <br />
+            Analysis ID: {summary.latest_snv_es_index?.id} <br />
+            {summary.latest_snv_es_index?.name}
+        </p>
+    )
+
+    const latestSvIndexSgCount = summary.latest_sv_es_index?.sg_count ?? 0
+    const percentageInSvIndex = calculatePercentage(
+        latestSvIndexSgCount,
+        summary.total_sequencing_groups
+    )
+    const inSvIndexTooltip = (
+        <p>
+            {latestSvIndexSgCount} / {summary.total_sequencing_groups} Sequencing Groups
+            <br />
+            Latest {summary.sequencing_type} SV Index <br />
+            {summary.latest_sv_es_index?.timestamp}
+            <br />
+            Analysis ID: {summary.latest_sv_es_index?.id} <br />
+            {summary.latest_sv_es_index?.name} <br />
+        </p>
+    )
+
+    switch (key) {
+        case 'aligned_percentage':
+            return (
+                <PercentageCell
+                    percentage={percentageAligned}
+                    tooltipContent={alignedCellTooltip}
+                    isDarkMode={isDarkMode}
+                    getPercentageColor={getPercentageColor}
+                ></PercentageCell>
+            )
+        case 'annotated_dataset_percentage':
+            return (
+                <PercentageCell
+                    percentage={percentageInJointCall}
+                    tooltipContent={inJointCallTooltip}
+                    isDarkMode={isDarkMode}
+                    getPercentageColor={getPercentageColor}
+                ></PercentageCell>
+            )
+        case 'snv_index_percentage':
+            return (
+                <PercentageCell
+                    percentage={percentageInSnvIndex}
+                    tooltipContent={inSnvIndexTooltip}
+                    isDarkMode={isDarkMode}
+                    getPercentageColor={getPercentageColor}
+                ></PercentageCell>
+            )
+        case 'sv_index_percentage':
+            return (
+                <PercentageCell
+                    percentage={percentageInSvIndex}
+                    tooltipContent={inSvIndexTooltip}
+                    isDarkMode={isDarkMode}
+                    getPercentageColor={getPercentageColor}
+                ></PercentageCell>
+            )
+        default:
+            return (
+                <SUITable.Cell className="table-cell">
+                    {summary[key as keyof ProjectInsightsSummary]}
+                </SUITable.Cell>
+            )
+    }
+}
+
+const SummaryTableRow: React.FC<{ summary: ProjectInsightsSummary }> = ({ summary }) => {
+    const theme = React.useContext(ThemeContext)
+    const isDarkMode = theme.theme === 'dark-mode'
+    const rowClassName = getRowClassName(summary.sequencing_type)
+    return (
+        <SUITable.Row
+            key={`${summary.dataset}-${summary.sequencing_type}-${summary.sequencing_technology}`}
```

Comment:
FWIW, you only need a key inside a .map within a function, and the key just has to be neighbourly unique (within a component).